template.yaml

```diff
@@ -324,7 +341,7 @@ Resources:
     Properties:
       ContainerDefinitions:
         - Essential: true
-          Image: CONTAINER-IMAGE-PLACEHOLDER
+          Image: 440208678480.dkr.ecr.eu-west-2.amazonaws.com/f2f-cri-front-ecr-containerrepository-lrazucdwjqvv:a4dafe8174341ab6d42863b029ac5bdf27fb36b4
```

Comment:
remove this change, is only needed for a test deploy and should not be committed to main